### ISSUE TITLE
code to produce Figure 5 in the paper 

### ISSUE THREAD

**Issue by tianyu0207** [shjung13/Standardized-max-logits#2]
Hi 

Thanks for the great work. I was looking at figure 5 - unexpected object detected with TPR 95, and I was just wondering could you provide the code to generate such figure. The figure looks amazing and I would like to use such a figure in my paper. Thanks so much for the help. 

![image](https://user-images.githubusercontent.com/19222962/140690427-79258167-0c91-4fe2-86a7-c8da93e40ece.png)

**Comment by shjung13** [comment]
Hello @tianyu0207, thank you for your interest in our work!
We currently don't have the visualization code for this figure. If I remember correctly, we obtained the threshold using scikit learn library and calculated the binary mask with that threshold.
For the blended images, we used the PIL blend function with an image and its corresponding GT color mask. 
I hope this would help you for solving your problem :)
